do.xml

```diff
@@ -18,6 +18,49 @@
 			<taxRule iso_code_country="do" id_tax="2" />
 		</taxRulesGroup>
 	</taxes>
+	<states>
+		<state name="Azua" iso_code="02" country="DO" zone="Central America/Antilla" />
+		<state name="Baoruco" iso_code="03" country="DO" zone="Central America/Antilla" />
+		<state name="Barahona" iso_code="04" country="DO" zone="Central America/Antilla" />
+		<state name="Cibao Nordeste" iso_code="33" country="DO" zone="Central America/Antilla" />
+		<state name="Cibao Noroeste" iso_code="34" country="DO" zone="Central America/Antilla" />
+		<state name="Cibao Sur" iso_code="36" country="DO" zone="Central America/Antilla" />
+		<state name="Dajabón" iso_code="05" country="DO" zone="Central America/Antilla" />
+		<state name="Distrito Nacional (Santo Domingo)" iso_code="01" country="DO" zone="Central America/Antilla" />
```

Comment:
Distrito Nacional (Santo Domingo)

It's more than 32 characters. Could we keep Santo Domingo only?